<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgAlpha</class>
 <widget class="QDialog" name="CDlgAlpha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>298</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>298</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="sImgBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>255</width>
     <height>4</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSlider" name="sldAlpha">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>255</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="sImgAlpha">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>58</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set the desired alpha value for drawing.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eAlpha">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>145</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>999; </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgAlpha</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgAlpha</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
